README.md

```diff
@@ -891,6 +891,7 @@ Table of Contents
   * [render.com](https://render.com) — A unified platform to build and run all your apps and web app free SSL, a global CDN, private networks and auto deploys from Git, free for static web page.
   * [sourceforge.net](https://sourceforge.net/) — Find, Create and Publish Open Source software for free
   * [Stormkit](https://stormkit.io/) — Integrate building, deploying and hosting seamlessly with your git flow of your JAMStack or Node.JS app. 50 GB bandwith and 10m requests for free per month including free SSL.
+  * [InfinitzHost](https://dash.infinitzhost.com/order/main/packages/Cpanel-Hosting/?group_id=4) — Free Hosting With PHP 7 & 8, MySQL, FTP, File Manager, E-Mails, free domain hosting, DNS Zone Editor, FREE Online Livechat support.
```

Comment:
while yes, technically this is a free service, I think its below the bar of things I am likely to keep accepting - and I think I will soon review this section and cull off others like this.

There's a massive churn of entries here, these kinds of rebranded cpanel based service are just not reliable and end up placing a burdon on the maintenance of the list.

Perhaps that doesnt describe you, but I think I'm going to blanket reject this kind of host.

Reply:
I can help with removing those fake(or rebranded) cPanel hosts tho, Shall I open a PR with that?

Reply:
I'd appreciate that for sure!